scripts/install_narrative.sh

```diff
@@ -124,6 +124,8 @@ then
         exit 1
     fi
 
+    # Install sklearn and clustergrammer
+    # ------------------------------
```

Comment:
adding this text so it is easier to compare `install_narrative` and `install_docker_narrative` to ensure they are doing (mostly) the same thing